lib/MusicBrainz/Server/Form/Admin/Attributes/Script.pm

```diff
@@ -27,9 +30,19 @@ has_field 'iso_number' => (
 );
 
 has_field 'frequency' => (
-    type => '+MusicBrainz::Server::Form::Field::Integer',
+    type => 'Select',
+    required => 1
 );
 
+sub options_frequency {
+    return [
+        $SCRIPT_FREQUENCY_HIDDEN, lp('Hidden', 'language_script_frequency'),
+        $SCRIPT_FREQUENCY_UNCOMMON, lp('Uncommon', 'language_script_frequency'),
+        $SCRIPT_FREQUENCY_OTHER, lp('Other', 'language_script_frequency'),
```

Comment:
`OTHER` should come before `UNCOMMON` to follow ascending order (as in constants).

Reply:
It was wrong in constants :) Fixed.